en-US/mozorg/about-2019.lang

```diff
@@ -165,7 +161,7 @@ Your voice. Your code. Your support. There are so many ways to join the fight fo
 
 # Link points to https://www.mozilla.org/contribute/
 ;Get Involved
-Get Involved
+Get InvInvolvedInvolvedolved
```

Comment:
Wut?

Reply:
This string is in main.lang, I intend to keep it here. Will fix it.

Reply:
Just to make sure: it's clear that I'm asking why you have "Get InvInvolvedInvolvedolved" as translation?